libraries/botbuilder-vendors/vendors/recognizers-text/recognizers-sequence/index.js

```
@@ -0,0 +1,602 @@
+              data: matchSource.has(srcMatch) ? matchSource.get(srcMatch) : null
+            });
+          }
+        }
+      } else {
+        lastNotMatched = i;
+      }
+    }
+    return results;
+  }
+  isLetterOrDigit(c) {
+    return new RegExp("[0-9a-zA-z]").test(c);
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)